docs/package.json

```diff
@@ -11,8 +11,8 @@
     "build:notfound": "node scripts/generate-notfound.js",
     "build:docs": "node scripts/generate-documentation.js",
     "build:blog": "node scripts/generate-blog.js",
-    "build": "yarn build:assets && yarn build:home && yarn build:blog && yarn build:docs && yarn build:notfound",
-    "watch": "cross-env WATCH=1 yarn start",
+    "build": "npm run build:assets && npm run build:home && npm run build:blog && npm",
```

Comment:
Why is the behavior changed here? (`yarn build:docs && yarn build:notfound` is removed.) Also I think `npm` does nothing.

Reply:
When running a `yarn deploy:docs` from the root repo, Heroku fails because we're pushing the `docs` as a subtree.  `docs` doesn't have a yarn.lock file, so Heroku doesn't recognize it as a yarn project.  When it goes to build and start, yarn isn't installed and it fails.

Reply:
`npm` is available in the Heroku build environment, though, so this fix gets everything to run.

Reply:
@tdalbo92 should answer that question